src/components/Preferences/ModalNav.vue

```diff
@@ -28,6 +28,7 @@ export default Vue.extend({
     <nav-item
       v-for="navItem in navItems"
       :key="navItem"
+      :data-test="`nav${navItem}`"
```

Comment:
Something like this would be better, but what we have works well enough:
```suggestion
      :data-test="'nav' + navItem.replace(/\s*\p{Lu}/gu, (c) => `-${c.toLowerCase()}`)"
```

Reply:
Sure, I introduced a method and replaced the regex with something that will hopefully be a little more easily understood at a glance. Let me know what you think